frontend/src/features/settings/account/PasswordForm.tsx

```diff
@@ -117,7 +120,13 @@ const PasswordForm = () => {
           />
         </Grid>
         <Grid item>
-          <Button type="submit" color="secondary" fullWidth variant="contained">
+          <Button
+            fullWidth
+            type="submit"
+            color="secondary"
+            variant="contained"
+            disabled={disabled}
```

Comment:
Unrelated to the PR, but would it be better with `diasbled || !passwordConfirmMatches` ?

Reply:
We could indeed disable the submit button if the whole form is not valid. In our case it means:
- it's not waiting for an HTTP response ;
- password and confirmation matches ;
- and the old password field is not empty.

This way the form will perform a full validation before enabling the button, which would be more user friendly.

Nevertheless, my personal preference would be to keep the form as is, because for me, disabled buttons appear like either disabled features, temporary unavailable features, or features with strong impact like deleting an account. I don't feel invited to use forms with disabled submit buttons, as I don't feel invited to delete my account on Tournesol. On the other hand, I feel invited to change my e-mail, my password, etc.

It's a really small preference, but still a preference : ) I'm curious about your opinion on this.

Reply:
My suggestion was a question full of doubts :) I understand and find your reasons convincing.